S/SuiteSparse@5/bundled/SuiteSparse_wrapper/Makefile

```diff
@@ -11,4 +11,4 @@ else
 endif
 
 install:
-	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -O2 -shared -fPIC -I $(prefix)/include SuiteSparse_wrapper.c -o $(prefix)/lib/libsuitesparse_wrapper.$(SHLIB_EXT) -L $(prefix)/lib $(RPATH) $(LDFLAGS) -lcholmod -lumfpack -lspqr
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -O2 -shared -fPIC -I $(prefix)/include -I ${WORKSPACE}/srcdir/SuiteSparse/include SuiteSparse_wrapper.c -o $(libdir)/libsuitesparse_wrapper.$(SHLIB_EXT) -L $(libdir) $(RPATH) $(LDFLAGS) -lcholmod -lumfpack -lspqr
```

Comment:
I've added `-I ${WORKSPACE}/srcdir/SuiteSparse/include` because for some reasons the header files are not being installed for Windows.  This is a temporary work around to make this compile, but probably we want to eventually install the header files, too.

Reply:
Yes, I think we will need the headers for Sundials to compile.

Reply:
This is interesting.  For some reasons header files are being installed to `/workspace/srcdir/SuiteSparse/include`, even though the SuiteSparse configuration summary says:

Reply:
Ok, setting `INSTALL_INCLUDE="${prefix}/include"` from the command line seems sufficient to convince it to install the header files to the right place.  I've removed the above work around.